docs/migration_guide.rst

```diff
@@ -1,3 +1,28 @@
+0.8.0 Migration guide
+=====================
+
+Cairo-lang 0.10.1 brings support for `DEPLOY_ACCOUNT` transactions that will completely
+replace currently used `DEPLOY` transactions sometime in the future. You should
+already modify your applications to use new deployment flow.
+
+1. Declare a contract on starknet using `Declare` transaction
+2. Pre-fund the address of new account with enough tokens to cover transaction costs
+3. Send a `DeployAccount` transaction to the pre-funded address
```

Comment:
We might also mention UDC here, linking to forum thread.

Reply:
@war-in is making a more in-depth guide in #464. Maybe only a mention of UDC would be enough here?

Reply:
I am creating a guide for the `DeployAccount` transaction mainly. I think that `migration_guide` is a better place to mention the UDC PR. I'll add it to the `Setup` once it will be ready